src/integrationTest/java/uk/gov/hmcts/darts/notification/service/NotificationServiceTest.java

```diff
@@ -0,0 +1,214 @@
+    }
+
+    @Test
+    void saveNotificationToDbMultipleEmails() {
+        String caseId = "saveNotificationToDbMultipleEmails";
+        SaveNotificationToDbRequest request = SaveNotificationToDbRequest.builder()
+            .eventId("An eventId")
+            .caseId(caseId)
+            .emailAddresses("test@test.com,test2@test.com")
+            .templateValues("a json string")
+            .build();
+        service.scheduleNotification(request);
+
+        List<Notification> resultList = notificationRepo.findByCaseId(caseId);
+        assertEquals(2, resultList.size(), "number of records is wrong.");
+        assertEquals(TEST_EMAIL_ADDRESS, resultList.get(0).getEmailAddress(), "first email address is wrong");
+        assertEquals("test2@test.com", resultList.get(1).getEmailAddress(), "");
+    }
+
+    @Test
+    void scheduleNotificationInvalidEmail() {
+        String caseId = "scheduleNotification";
+        SaveNotificationToDbRequest request = SaveNotificationToDbRequest.builder()
+            .eventId("An eventId")
+            .caseId(caseId)
+            .emailAddresses("test@test@.com")
+            .templateValues("a json string")
+            .build();
+        service.scheduleNotification(request);
+        List<Notification> resultList = notificationRepo.findByCaseId(caseId);
+        assertEquals(0, resultList.size());
+    }
+
+    @Test
+    void sendNotificationToGovNotify() throws TemplateNotFoundException {
+        when(templateIdHelper.findTemplateId(REQUEST_TO_TRANSCRIBER_TEMPLATE_NAME)).thenReturn(
+            "976bf288-705d-4cbb-b24f-c5529abf14cf");
+
+        String caseId = "sendNotificationToGovNotify";
+        SaveNotificationToDbRequest request = SaveNotificationToDbRequest.builder()
+            .eventId(REQUEST_TO_TRANSCRIBER_TEMPLATE_NAME)
+            .caseId(caseId)
+            .emailAddresses(TEST_EMAIL_ADDRESS)
+            .templateValues("{\n" +
+                            "  \"key1\": \"value1\",\n" +
+                                "  \"key2\": \"value2\",\n" +
+                                "  \"key3\": \"value3\",\n" +
+                                "  \"key4\": \"value4\",\n" +
+                                "  \"key5\": \"value5\"\n" +
+                                "}")
+            .build();
+        service.scheduleNotification(request);
+        service.sendNotificationToGovNotify();
+        List<Notification> resultList = notificationRepo.findByCaseId(caseId);
+        Notification result = resultList.get(0);
+        assertEquals("SENT", result.getStatus(), "Object may not have sent");
+
+    }
+
+    @Test
+    void sendNotificationToGovNotifyInvalidTemplateId(TestInfo testInfo) throws TemplateNotFoundException, NotificationClientException {
+        when(templateIdHelper.findTemplateId(REQUEST_TO_TRANSCRIBER_TEMPLATE_NAME)).thenReturn(
+            "976bf288-1234-1234-1234-c5529abf14cf");//invalid tempalte number
```

Comment:
spelling in comment. Do we really need the comment? Instead, maybe you could just change the template ID to something so it is clear e.g. "INVALID-TEMPLATE-ID" or something.